### DISCUSSION TITLE
Youtube links sometimes dont work

### DISCUSSION THREAD

**Discussion by t3rry327** [SudhanPlayz/Discord-MusicBot#561] [Q&A] [answered]
**Please describe the problem you are having in as much detail as possible:**
Sometimes the bot cant find youtube links

**Include a reproducible code sample here, if possible:**
https://imgur.com/a/02hUhDu

**Further details:**

- discord.js version: 12.5.3
- Node.js version: 14.18.1
- Operating system: ubuntu server 18.04 lts
- Priority this issue should have – please be realistic and elaborate if possible: medium

**Comment by captainion2119** [comment]
hmm, this might be a lavalink error, cuz sometimes the vids that are requested have some kind copyright lock (this is a theory, cuz i faced the same problem), or somethin.

if u play the same song (using "*prefix* *song* lyrics" adding the word lyrics at the end.) it plays the song.

lets see what changes are made for this, but I believe its a lavalink side error.

**Comment by Coponte** [comment]
YouTube is blocking most reguest of getting video material. How exactly ? I dont know :)

**Comment by JotaroKujo0525** [comment]
0



> **Please describe the problem you are having in as much detail as possible:** Sometimes the bot cant find youtube links
> 
> **Include a reproducible code sample here, if possible:** https://imgur.com/a/02hUhDu
> 
> **Further details:**
> 
> * discord.js version: 12.5.3
> * Node.js version: 14.18.1
> * Operating system: ubuntu server 18.04 lts
> * Priority this issue should have – please be realistic and elaborate if possible: medium

Try using the `youtu.be` link instead of that
Like this `https://youtu.be/SRWrQMwGYsQ`

**Comment by t3rry327** [comment]
im using lava.link server
i will host my own lavalink server and try..

(update tomorrow)

**Comment by JotaroKujo0525** [comment]
> im using lava.link server i will host my own lavalink server and try..
> 
> (update tomorrow)

Mate, even if you did try that. That `youtube.com` smth link sometimes just don't work. Use the `youtu.be` link instead

**Comment by t3rry327** [comment]
never.

**Comment by JotaroKujo0525** [comment]
Then don't expect that your problem will be fixed ;-;

**Comment by captainion2119** [comment]
> > im using lava.link server i will host my own lavalink server and try..
> > 
> > (update tomorrow)
> 
> Mate, even if you did try that. That `youtube.com` smth link sometimes just don't work. Use the `youtu.be` link instead

This. ^

**Comment by DarrenOfficial** [comment]
> > im using lava.link server i will host my own lavalink server and try..
> > (update tomorrow)
> 
> Mate, even if you did try that. That `youtube.com` smth link sometimes just don't work. Use the `youtu.be` link instead

Those are essentially the same, youtu.be is just a redirect to youtube.com

**Comment by DarrenOfficial** [comment, marked as answer]
> im using lava.link server i will host my own lavalink server and try..
> 
> (update tomorrow)

And lava.link is prone to have those issue, I recommend switching to another host, here's a list of lavalink that you can try https://lavalink.darrennathanael.com
